helpers/tests/pathmap/test_pathmap.py

```diff
@@ -1,86 +1,68 @@
-from helpers.pathmap import (
-    Tree,
-    _check_ancestors,
-    _extract_match,
-    _resolve_path,
-    clean_path,
-)
-
-# ========== Mock data ===========
-before = [
-    "not/found.py",
-    "/Users/user/owner/repo/src/components/login.js",
-    "site-packages/package/__init__.py",
-    "path.py",
-    "a/b/../Path With\\ Space",
-]
-
-after = [
-    None,
-    "src/components/login.js",
-    "package/__init__.py",
-    "path.py",
-    "a/Path With Space",
-]
-
-toc = ["" if x is None else x for x in after]
-
-
-# ========= END Mock data ==========
+from helpers.pathmap import Tree, _check_ancestors, _clean_path
+
+
 def test_clean_path():
     path = "**/some/directory"
-    assert clean_path(path) == "some/directory"
+    assert _clean_path(path) == "some/directory"
     path = "some/path\r/with/tabs\r"
-    assert clean_path(path) == "some/path/with/tabs"
+    assert _clean_path(path) == "some/path/with/tabs"
     path = "some\ very_long/directory\ name"
-    assert clean_path(path) == "some very_long/directory name"
+    assert _clean_path(path) == "some very_long/directory name"
     path = "ms\\style\\directory"
-    assert clean_path(path) == "ms/style/directory"
-
-
-def test_extract_match():
-    toc = ",src/components/login.js,"
-    index = toc.find("components")
-    extracted = _extract_match(toc, index)
-    assert extracted == "src/components/login.js"
+    assert _clean_path(path) == "ms/style/directory"
 
 
 def test_resolve_path():
-    # short to long
-    path = "Src/components/login.js"
+    expected_path = "src/components/login.js"
     tree = Tree()
-    tree.construct_tree(toc)
-    new_path = _resolve_path(tree, path)
-    assert new_path == "src/components/login.js"
+    tree.construct_tree([expected_path])
+
+    assert tree.resolve_path("Src/components/login.js") == expected_path
 
 
 def test_resolve_case():
     tree = Tree()
     tree.construct_tree(["Aa/Bb/cc", "Aa/Bb/Cc"])
-    assert _resolve_path(tree, "aa/bb/cc") == "Aa/Bb/cc"
-    assert _resolve_path(tree, "aa/bb/Cc") == "Aa/Bb/Cc"
+    assert tree.resolve_path("aa/bb/cc") == "Aa/Bb/cc"
+    assert tree.resolve_path("aa/bb/Cc") == "Aa/Bb/Cc"
 
 
 def test_resolve_paths():
+    before = [
+        "not/found.py",
+        "/Users/user/owner/repo/src/components/login.js",
+        "site-packages/package/__init__.py",
+        "path.py",
+        "a/b/../Path With\\ Space",
+    ]
+
+    after = [
+        None,
+        "src/components/login.js",
+        "package/__init__.py",
+        "path.py",
+        "a/Path With Space",
+    ]
```

Comment:
I would suggest using pytest [parametrize](https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids). With parametrize, the test will run all cases even if one fails.

Reply:
Thanks for the suggestion. In this particular case, `after` serves a dual purpose, initializing the tree, and serving as assertion, so I would rather skip it in this particular case.